configs/typescript-config/tsconfig.reactnative.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Rename this file to `tsconfig.react-native.json`

Reply:
Done